.travis.yml

```diff
@@ -15,7 +15,7 @@ script:
   - |
     if [[ "$TEST_TYPE" == "docker" ]]; then
     # run tests
-      docker exec -it app /bin/bash -c ". $BASHRC && cd $DOCKER_WORKING_DIR/reg_tests && python run_reg_tests.py -nodiff"
+      docker exec -it app /bin/bash -c ". $BASHRC && cd $DOCKER_WORKING_DIR/tests && testflo . -v -n 1"
```

Comment:
I think we can run testflo with 2 procs, even though the tests themselves are set to run on a single proc. However, this might break some tests if they are trying to read/write the same files because this way, testflo would run 2 tests at a time, using a single processor for each test. If the tests do not write files that are used by other tests concurrently, it should be fine and we can use 2 procs, but @nwu63 would know more as the expert

Reply:
The only test that writes a file that I know of is Nick's cylinder test which writes an xyz file. No other test writes that file. There are some simultaneous reads. I think it's probably fine. Will this break reg tests for users who don't have mpi4py installed and configured correctly?

Reply:
In general for these test we should try to have unique filenames for each test. The file read should be fine. As @anilyil mentioned we should probably use `-n 2` here to try to accelerate testing on Travis since all test use `N_PROCS = 1` anyways. This is just for running with testflo on Travis so this does not break anything. 
If I recall correctly testflo will run even if mpi is not installed. The `--nompi` flag can also be supplied to not use mpi (for those who are configured to run in parallel). Also, if the user does not have testflo installed one can always just run the tests manually also.
Finally, please remember to update run command in the [runscript in the docker repo](https://github.com/mdolab/docker/blob/master/extras/master_test_only_updated_GFORTRAN.py) for the collective testing.

Reply:
I doubt it will speed up Travis, since what we actually get is 1 physical proc with 2 logical procs. I am in favour of keeping it `-n 1` here. On the build bot we can increase to `-n 4` since there are more procs available.